<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLinksMenu</class>
 <widget class="QDialog" name="AddLinksMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_addLink">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Link</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_PID">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_PID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Person ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_CID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_CID">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pLAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pLBack">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="p_error">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="c_error">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_person">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table_comps">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="l_person_table">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>People</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_comps_table">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>71</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Computers</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
